modules/homepage/src/main/frontend/src/themePage/QuickSearchResults.jsx

```diff
@@ -60,9 +60,9 @@ function QuickSearchResults(props) {
       <Card>
         <CardHeader
           title={
-            <Button className={classes.quickSearchResultsTitle}>
+            <Typography variant="h6">
```

Comment:
Let's use h4 instead of h6 like we do in the admin screens.

```suggestion
            <Typography variant="h4">
```